operator/pkg/reconciliation/reconcile_racks.go

```diff
@@ -476,6 +476,9 @@ func (rc *ReconciliationContext) CheckPodsReady(endpointData httphelper.CassMeta
 	desiredSize := int(rc.Datacenter.Spec.Size)
 
 	if desiredSize == readyPodCount && desiredSize == startedLabelCount {
+		if nodeStarted {
```

Comment:
I think you should add a one line comment here

also why not move this directly under
```
	nodeIsStarting, nodeStarted, err := rc.findStartingNodes()
```
?

Reply:
comment added.

I hadn't really considered moving the `if` block. That will require some further discussion to ensure I don't wind up introducing some regressions.

Reply:
yeah let's not introduce instability, I can tackle this when refactoring some stuff in an upcoming PR